spec/hosts/nodes/lsstcam-vw03.ls.lsst.org_spec.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe 'lsstcan-vw03.ls.lsst.org', :sitepp do
```

Comment:
```suggestion
describe 'lsstcam-vw03.ls.lsst.org', :sitepp do
```

Reply:
Thanks! The n and m keys are pretty close! 🤣